package.json

```diff
@@ -29,7 +29,7 @@
   "homepage": "https://github.com/alpheios-project/wordsvc#readme",
   "devDependencies": {
     "alpheios-client-adapters": "github:alpheios-project/client-adapters",
-    "alpheios-components": "github:alpheios-project/components",
+    "alpheios-components": "github:alpheios-project/components#comp-i341-toolbar-integration",
```

Comment:
branch should be removed, I think

Reply:
Oh, that's correct. Thanks for catching this. Will update